### PR TITLE
CW-86 Create GitHub Prod workflow.

### PR DESCRIPTION
**Pull request by 108ddy**

### CODE REVIEW

File: .github/workflows/prod.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - run: mv Dockerfile.production Dockerfile
+      - uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.IMAGE_NAME }}
+  remote-host-load-image:
```
Comment (nikitamelnik1504) [inline .github/workflows/prod.yml:30]: Please remove second job for now.

File: src/views/page/CreateWord/index.vue [hunk 2]
```diff
@@ -78,7 +78,7 @@
                       type="button"
                       class="minus w-25 h-100 position-absolute start-0 d-flex justify-content-center align-items-center"
                       :class="{ disabled: +asset.value === 0.1 }"
-                      :disabled="((+asset.value === 0.1) | playRunning)"
+                      :disabled="(+asset.value === 0.1) | playRunning"
```
Comment (nikitamelnik1504) [inline src/views/page/CreateWord/index.vue]: ?

**Review by nikitamelnik1504 — CHANGES_REQUESTED** [review]

File: src/views/page/CreateWord/index.vue (hunk 2 above)
Comment (108ddy) [reply src/views/page/CreateWord/index.vue]: eslint make this

Comment (nikitamelnik1504) [reply src/views/page/CreateWord/index.vue]: Does your site env works now?

**Review by nikitamelnik1504 — APPROVED** [review]

**Pull request merged**